rpm/maep-qt.spec

```diff
@@ -69,6 +69,7 @@ install -m 644 -p /usr/lib/libfreetype.so.6 %{buildroot}/usr/share/%{name}/lib/
 install -m 644 -p /usr/lib/libicui18n.so.52 %{buildroot}/usr/share/%{name}/lib/
 install -m 644 -p /usr/lib/libicuuc.so.52 %{buildroot}/usr/share/%{name}/lib/
 install -m 644 -p /usr/lib/libicudata.so.52 %{buildroot}/usr/share/%{name}/lib/
+install -m 644 -p /lib/ld-linux.so.? %{buildroot}/usr/share/%{name}/lib/
 
```

Comment:
Without these changes I get the following error message

```
Requires
========
ERROR [ld-linux.so.2] Dependency not allowed
ERROR [ld-linux.so.2(GLIBC_2.3)] Dependency not allowed
ERROR [libdl.so.2(GLIBC_2.1)] Cannot require shared library: 'libdl.so.2(GLIBC_2.1)'
INFO [harbour-maep-qt] Please see our FAQ here: https://harbour.jolla.com/faq#2.6.0 how to use '__provides_exclude_from' and '__requires_exclude' .spec file to avoid that
FAILED
```
when building the rpm package and no package is built. I am using Qt Creator 3.5.0 (SailfishOS) Revision d2fe010 on Linux.